src/index.ts

```diff
@@ -42,6 +42,7 @@ export { SlideshowHero } from 'SlideshowHero/SlideshowHero';
 export { SocialShare } from 'SocialShare/SocialShare';
 export { TableauChart } from 'TableauChart/TableauChart';
 export { Text } from 'Text/Text';
+export { default as TextArea } from 'TextArea';
```

Comment:
Just out of interest, why have you done it this way instead of `... from TextArea/TextArea`?

Reply:
_(copy and pasted the below from an earlier PR where I'd done the same thing)_

This is a slightly different method of exporting, but I think it's preferable and the other exports in this file should follow the same pattern:

- makes easier to change components' main export without touching this file (e.g. if SiteAlert/index.ts wants to `export default SiteAlertView` rather than `export default SiteAlert`, then we would not need to touch this file 🚀 
- prevents a linting error on the SiteAlert/index.ts because the export is never used (not imported elsewhere in the app)